cmd.js

```diff
@@ -85,6 +90,28 @@ async function cmd (argv, banner = defaultBanner) {
   args.argv = subprocessArgv
   args.pathToNodeBinary = pathToNodeBinary
 
+  if (typeof args.sourceMaps === 'undefined') {
+    args.sourceMaps = null
+  } else {
+    if (args.sourceMaps !== '') {
+      const sourceMapFile = resolve('.', args.sourceMaps)
+      if (fs.existsSync(sourceMapFile)) {
+        try {
+          args.sourceMaps = JSON.parse(fs.readFileSync(sourceMapFile))
+        } catch (err) {
+          console.error(`Couldn't parse source map file ${sourceMapFile}, is it valid json?`)
+        }
+      }
+    }
+    if (typeof args.sourceMaps !== 'object') {
+      args.sourceMaps = {}
+    }
```

Comment:
That is a bit to much magic for me. I suggest to either throw an error or to remove the `if` so a regular error would occur later on.

Reply:
The magic here is with regards to validation with schema.json. The argument from the cli will always be a string, so we have to make sure its an object because otherwise ajv will complain :)